src/Utils/AffineTransform.vala

```diff
@@ -45,6 +48,32 @@ public class Akira.Utils.AffineTransform : Object {
         var delta_x = fix_size (event_x - initial_event_x);
         var delta_y = fix_size (event_y - initial_event_y);
 
+        bool snap_x = false;
+        bool snap_y = false;
+
+        if (data.horizontal.wants_snap() && delta_x.abs() <= 4) {
+            if (data.horizontal.type == SnapManager.MatchType.FUZZY) {
+                delta_x = data.horizontal.snap_offset();
+                snap_x = true;
+            }
+            else {
+                delta_x = 0;
+                snap_x = true;
+            }
+        }
```

Comment:
Interesting, I think this is looking like a good solid approach.
We can rewrite this to be a bit shorter like:
```PHP
if (data.horizontal.wants_snap() && delta_x.abs() <= 4) {
    delta_x = data.horizontal.type == SnapManager.MatchType.FUZZY ? data.horizontal.snap_offset() : 0;
    snap_x = true;
}
```
And same thing for the vertical condition.

Reply:
Also, we should get the `SENSITIVITY` from the SnapManager instead of manually writing it, so we're safe if that value changes for whatever reason.